src/main/webapp/WEB-INF/tags/instructor/studentList/studentsTable.tag

```diff
@@ -5,7 +5,7 @@
 <%@ attribute name="index" required="true" %>
 <c:set var="PANEL_TYPE" value="${course.courseArchived ? 'panel-default' : 'panel-info'}" />
 <div class="panel ${PANEL_TYPE}">
-    <div class="panel-heading ajax_submit">
+    <div class="panel-heading ajax_submit" onclick="toggleChevron(this)">
```

Comment:
@amarlearning you should implement a solution that does not involve any JS inside the JSP/tag/HTML files